ChangeLog.md

```diff
@@ -2,6 +2,10 @@
 
 ## 2.0.1-dev (changes since 2.0.1 go here)
 
+ * A unified API for Gibb's sampling (including the previous ``_gibbs`` and 
```

Comment:
Generally we don't reference private functions in the changelog since it's user facing (the functionality is private, so it's not a user-facing change). I would make this:

*A unified API for sourcetracking with gibbs sampling, including leave-one-out cross-validation, has been created and is accessible as ``sourcetracker.gibbs``.